packages/block-editor/src/components/rich-text/event-listeners/paste-handler.js

```diff
@@ -92,15 +90,7 @@ export default ( props ) => ( element ) => {
 
 		let mode = 'INLINE';
 
-		const trimmedPlainText = plainText.trim();
-
-		if (
-			__unstableEmbedURLOnPaste &&
```

Comment:
Hi @ellatrix, could you provide more context on why we are doing these changes and its relation to the pasting SVG?